{"uid":"480d6514140b7422","name":"Successful Login","fullName":"Login functionallity: Successful Login","historyId":"6eb4afeab05f2b214d8d7d19c20f537e","time":{"start":1733550093217,"stop":1733550100507,"duration":7290},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//H1[text()='Today']\"}\n  (Session info: chrome=131.0.6778.70); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF694D02775+28773]\n\t(No symbol) [0x00007FF694C6AFB0]\n\t(No symbol) [0x00007FF694B0552A]\n\t(No symbol) [0x00007FF694B58EAE]\n\t(No symbol) [0x00007FF694B5919C]\n\t(No symbol) [0x00007FF694BA23F7]\n\t(No symbol) [0x00007FF694B7EFDF]\n\t(No symbol) [0x00007FF694B9F1A4]\n\t(No symbol) [0x00007FF694B7ED43]\n\t(No symbol) [0x00007FF694B4A548]\n\t(No symbol) [0x00007FF694B4B6B1]\n\tGetHandleVerifier [0x00007FF69502F45D+3358029]\n\tGetHandleVerifier [0x00007FF69504430D+3443709]\n\tGetHandleVerifier [0x00007FF6950383FD+3394797]\n\tGetHandleVerifier [0x00007FF694DC929B+842635]\n\t(No symbol) [0x00007FF694C7654F]\n\t(No symbol) [0x00007FF694C71FA4]\n\t(No symbol) [0x00007FF694C7213D]\n\t(No symbol) [0x00007FF694C61629]\n\tBaseThreadInitThunk [0x00007FFD5C867BD4+20]\n\tRtlUserThreadStart [0x00007FFD5DD4CED1+33]\n\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/ui/steps/login_steps.py\", line 23, in step_verify_login_successful\n    assert context.login_page.driver.find_element(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//H1[text()='Today']\"}\n  (Session info: chrome=131.0.6778.70); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF694D02775+28773]\n\t(No symbol) [0x00007FF694C6AFB0]\n\t(No symbol) [0x00007FF694B0552A]\n\t(No symbol) [0x00007FF694B58EAE]\n\t(No symbol) [0x00007FF694B5919C]\n\t(No symbol) [0x00007FF694BA23F7]\n\t(No symbol) [0x00007FF694B7EFDF]\n\t(No symbol) [0x00007FF694B9F1A4]\n\t(No symbol) [0x00007FF694B7ED43]\n\t(No symbol) [0x00007FF694B4A548]\n\t(No symbol) [0x00007FF694B4B6B1]\n\tGetHandleVerifier [0x00007FF69502F45D+3358029]\n\tGetHandleVerifier [0x00007FF69504430D+3443709]\n\tGetHandleVerifier [0x00007FF6950383FD+3394797]\n\tGetHandleVerifier [0x00007FF694DC929B+842635]\n\t(No symbol) [0x00007FF694C7654F]\n\t(No symbol) [0x00007FF694C71FA4]\n\t(No symbol) [0x00007FF694C7213D]\n\t(No symbol) [0x00007FF694C61629]\n\tBaseThreadInitThunk [0x00007FFD5C867BD4+20]\n\tRtlUserThreadStart [0x00007FFD5DD4CED1+33]\n\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/ui/steps/login_steps.py\", line 23, in step_verify_login_successful\n    assert context.login_page.driver.find_element(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Given the user navigates to the login page","time":{"start":1733550093217,"stop":1733550094512,"duration":1295},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When the user logs in with valid credentials","time":{"start":1733550094512,"stop":1733550095351,"duration":839},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then the user should be redirected to the dashboard","time":{"start":1733550095351,"stop":1733550100506,"duration":5155},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//H1[text()='Today']\"}\n  (Session info: chrome=131.0.6778.70); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF694D02775+28773]\n\t(No symbol) [0x00007FF694C6AFB0]\n\t(No symbol) [0x00007FF694B0552A]\n\t(No symbol) [0x00007FF694B58EAE]\n\t(No symbol) [0x00007FF694B5919C]\n\t(No symbol) [0x00007FF694BA23F7]\n\t(No symbol) [0x00007FF694B7EFDF]\n\t(No symbol) [0x00007FF694B9F1A4]\n\t(No symbol) [0x00007FF694B7ED43]\n\t(No symbol) [0x00007FF694B4A548]\n\t(No symbol) [0x00007FF694B4B6B1]\n\tGetHandleVerifier [0x00007FF69502F45D+3358029]\n\tGetHandleVerifier [0x00007FF69504430D+3443709]\n\tGetHandleVerifier [0x00007FF6950383FD+3394797]\n\tGetHandleVerifier [0x00007FF694DC929B+842635]\n\t(No symbol) [0x00007FF694C7654F]\n\t(No symbol) [0x00007FF694C71FA4]\n\t(No symbol) [0x00007FF694C7213D]\n\t(No symbol) [0x00007FF694C61629]\n\tBaseThreadInitThunk [0x00007FFD5C867BD4+20]\n\tRtlUserThreadStart [0x00007FFD5DD4CED1+33]\n\n","statusTrace":"  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/ui/steps/login_steps.py\", line 23, in step_verify_login_successful\n    assert context.login_page.driver.find_element(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n  File \"/home/runner/.cache/pypoetry/virtualenvs/todoist-test-Iz2mgPfl-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":3,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"login"},{"name":"tag","value":"successful_login"},{"name":"feature","value":"Login functionallity"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["successful_login","login"]},"source":"480d6514140b7422.json","parameterValues":[]}